examples/language/lib/functions/getters_setters.dart

```diff
@@ -0,0 +1,42 @@
+/// Defines a private variable secret with `_`.
```

Comment:
I got slightly confused reading this comment as I wasn't sure if "secret" was referring to the variable name or the visibility of the variable.

Perhaps we can be more explicit by referencing the full name of the variable and add more details about the underscore?

Would something like this capture your intent?

```
// Defines a variable `_secret` that is private to the library since
// its identifier starts with an underscore (`_`).
```